### PR TITLE
pages.pl/*: update and add missing alias pages

### PR DESCRIPTION
**Pull request by acuteenvy**

### CODE REVIEW

**Comment by acuteenvy** [comment]
#14824 should fix the bot comments.

File: pages.pl/common/brave.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# brave
+
+> To polecenie jest aliasem `chromium`.
+> More information: <https://support.brave.com/hc/en-us/articles/360044860011-How-Do-I-Use-Command-Line-Flags-in-Brave>.
```
Comment (spageektti) [inline pages.pl/common/brave.md:4]:
```suggestion
> Więcej informacji: <https://support.brave.com/hc/en-us/articles/360044860011-How-Do-I-Use-Command-Line-Flags-in-Brave>.
```

File: pages.pl/common/opera.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# opera
+
+> To polecenie jest aliasem `chromium`.
+> More information: <https://opera.com>.
```
Comment (spageektti) [inline pages.pl/common/opera.md:4]:
```suggestion
> Więcej informacji: <https://opera.com>.
```

File: pages.pl/common/vivaldi.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# vivaldi
+
+> To polecenie jest aliasem `chromium`.
+> More information: <https://vivaldi.com>.
```
Comment (spageektti) [inline pages.pl/common/vivaldi.md:4]:
```suggestion
> Więcej informacji: <https://vivaldi.com>.
```

File: pages.pl/linux/mount.smb3.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# mount.smb3
+
+> To polecenie jest aliasem `mount.cifs`.
+> Uwaga: do wersji SMB poniżej 3 musisz użyć `mount.cifs`.
```
Comment (spageektti) [inline pages.pl/linux/mount.smb3.md:4]:
```suggestion
> Uwaga: dla wersji SMB poniżej 3 musisz użyć `mount.cifs`.
```
I am not sure I havent seen english page

**Review by spageektti — APPROVED** [review]

**Pull request merged**